starter-code/src/functions-and-arrays.js

```diff
@@ -11,14 +22,58 @@ var words = [
   'crackpot'
 ];
 
+function findLongestWord(words) {
+  console.log("Arguments: ", arguments);
+  console.log("Passed parameters: ", words);
+
+  if(words.length == 0) {
+    return undefined;
+  }
+
+  let length = 0;
+  let longest = [];
+
+  for(let i=0; i < words.length; i++){
+    if(words[i].length > length){
+        length = words[i].length;
```

Comment:
good logic! Though where are you using your variable length? Remember if you're not going to use it, you don't need to have a variable there